nbt-extra-kotlin/src/main/kotlin/net/kyori/adventure/nbt/extra/kotlin/ListBinaryTagConverters.kt

```diff
@@ -0,0 +1,47 @@
+package net.kyori.adventure.nbt.extra.kotlin
+
+import net.kyori.adventure.nbt.BinaryTag
+import net.kyori.adventure.nbt.BinaryTagTypes
+import net.kyori.adventure.nbt.ByteArrayBinaryTag
+import net.kyori.adventure.nbt.ByteBinaryTag
+import net.kyori.adventure.nbt.DoubleBinaryTag
+import net.kyori.adventure.nbt.FloatBinaryTag
+import net.kyori.adventure.nbt.IntArrayBinaryTag
+import net.kyori.adventure.nbt.IntBinaryTag
+import net.kyori.adventure.nbt.ListBinaryTag
+import net.kyori.adventure.nbt.LongArrayBinaryTag
+import net.kyori.adventure.nbt.LongBinaryTag
+import net.kyori.adventure.nbt.ShortBinaryTag
+import net.kyori.adventure.nbt.StringBinaryTag
+
+public fun <T : BinaryTag> List<T>.toBinaryTag(): ListBinaryTag = ListBinaryTag.from(this)
+
+@JvmName("toBinaryTagByte")
+public fun List<Byte>.toBinaryTag(): ListBinaryTag = ListBinaryTag.of(BinaryTagTypes.BYTE, map(ByteBinaryTag::of))
+
+@JvmName("toBinaryTagShort")
```

Comment:
```suggestion
@JvmName("toShortBinaryTag")
```
... etc (applies to all cases like this)